documentation/disaster-recovery.md

```diff
@@ -0,0 +1,59 @@
+# Disaster recovery
+
+## Data loss
+
+In the case of a data loss, we need to recover the data as soon as possible in order to resume normal service.
+The application's database is a postgres instance, which resides on PaaS. This provides a point-in-time backup with
+the resolution of 1 second, available between 5min and 7days ago - this is not applicable to postgres `tiny` plans. We can use
+terraform to create a new database using a point-in-time backup of the affacted database instance.
+
+### Make note of database failure time
+
+Make note of the time the database failure occurred, and then use this to calculate when the integrity of the data in the database was still viable. For instance,
+if data loss or corruption happened at 1200hrs, workout what snapshot time is best for the product. This would determine the value of `SNAPSHOT_TIME` env.
+
+### Freeze pipeline
+
+If not already communicated, no one should merge to master branch
+
+### Maintenance mode
+
+In the instance of data loss, if the application is unavailable, the offline site may be displayed.
+If the application is still available and there is a risk of users adding data, enable the maintenace mode, by setting [Maintenance mode](maintenance-mode.md).
+
+### Get postgres database ID
+
+Use the makefile's `get-postgres-instance-guid` to get the database guid, use the following command:
```

Comment:
```suggestion
Use the makefile's `get-postgres-instance-guid` to get the database guid, use the following command (set `CONFIRM_PRODUCTION` to `true`):
```

Reply:
not sure I follow....?

Reply:
I don't want people to copy and paste this command during a simulation exercise, and accidentally blow up production. They should write `true` themselves.